utils/config_generator.py

```diff
@@ -123,7 +124,10 @@ def update_config(
     experiment["trg"] = target
     experiment["bicleaner"]["dataset-thresholds"] = {}
     # Use separate vocabs for languages with different scripts
-    experiment["spm-vocab-split"] = get_default_script(source) != get_default_script(target)
+
+    experiment["spm-vocab-split"] = not (
```

Comment:
Why was this changed? The logic was to use split vocabs if the languages don't share the same script. How is it related to the script being phonemic?

Reply:
Let's discuss this in chat.

Reply:
Well, the idea was to use the same method for Korean as well (split vocabs 32k), it's just the improvement was not big enough to rerun distillation, so I'm not sure if we should make an exception for it.